asmcli/asmcli.sh

```diff
@@ -1486,7 +1541,10 @@ EOF
 validate_cluster() {
   local RESULT; RESULT=""
 
-  info "Confirming cluster information for ${PROJECT_ID}/${CLUSTER_LOCATION}/${CLUSTER_NAME}..."
+  local CURRENT_CONTEXT="$(kubectl config current-context)"
+  info "Confirming cluster information for ${CURRENT_CONTEXT}"
+  IFS="_" read GKE_PREFIX PROJECT_ID CLUSTER_LOCATION CLUSTER_NAME <<< "${CURRENT_CONTEXT}"
```

Comment:
For compatibility, can we use a heredoc instead of the bash specific herestring? e.g.

```
IFS="_" read -r GKE_PREFIX PROJECT_ID CLUSTER_LOCATION CLUSTER_NAME <<EOF
$(kubectl config current-context)
EOF
```

Reply:
done!